nix/mobile/android/build.nix

```diff
@@ -17,6 +17,7 @@
   hermesEnabled ? lib.getEnvWithDefault "ORG_GRADLE_PROJECT_hermesEnabled" "true",
   buildUrl ? lib.getEnvWithDefault "ORG_GRADLE_PROJECT_buildUrl" null,
   statusGoSrcOverride ? lib.getEnvWithDefault "STATUS_GO_SRC_OVERRIDE" null,
+  reactMetroPort ? lib.getEnvWithDefault "RCT_METRO_PORT" 8081,
```

Comment:
Why would you need this? Android derivation build run in a sandbox that has no network access. This is pointless.

Reply:
The intention is to forward the environment variable for `RCT_METRO_PORT` to the Gradle build step. This is because the react-native bindings will use the project property `reactNativeDevServerPort`, and I think this property needs to be defined at build time so that the react-native code knows which port to use when it has network access.

In comparison, when I didn't forward this environment variable, and I would run `make run-android`, the all the other scripts would use the correct `RCT_METRO_PORT` override, but the Android app would somehow still default to `8081`.

Reply:
Interesting, but I thought if you use Metro with `make run-metro` then we don't use the APK built by Nix.

Reply:
Oh I see, we do actually call `build-android.sh` inside of `/home/jakubgs/work/status-mobile/scripts/run-android.sh`:
https://github.com/status-im/status-mobile/blob/9f412577d732a2c51e791c66955b57a42c26685a/scripts/run-android.sh#L12
And that is then started:
https://github.com/status-im/status-mobile/blob/9f412577d732a2c51e791c66955b57a42c26685a/scripts/run-android.sh#L15-L17
That's why it matters what is set for APK built by the derivation.